src/main/java/core/basesyntax/figures/Square.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax.figures;
+
+import core.basesyntax.Figure;
+
+public class Square extends Figure {
+    private static final String type = "square";
+    private final int side;
+
+    public Square(int side, String color) {
+        this.side = side;
+        this.color = color;
```

Comment:
The constructor of the 'Square' class is missing a call to the superclass constructor. You need to call 'super(color);' to properly set the color in the 'Figure' superclass.